src/app/profile/my-spaces/my-spaces-search-dialog/my-spaces-search-spaces-dialog-space-item/my-spaces-search-spaces-dialog-space-item.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import {
+  initContext,
+  TestContext
+} from 'testing/test-context';
+
+import { MySpacesSearchSpacesDialogSpaceItemComponent } from './my-spaces-search-spaces-dialog-space-item.component';
+
+@Component({
+  template: '<my-spaces-search-spaces-dialog-space-item></my-spaces-search-spaces-dialog-space-item>'
+})
+class HostComponent { }
+
+describe('MySpacesSearchSpacesDialogSpaceItemComponent', () => {
+
+  type TestingContext = TestContext<MySpacesSearchSpacesDialogSpaceItemComponent, HostComponent>;
+  initContext(MySpacesSearchSpacesDialogSpaceItemComponent, HostComponent, {
+    imports: [ RouterTestingModule ]
+  });
+
+  it('should be instantiable', function(this: TestingContext): void {
+    expect(this.testedDirective).toBeDefined();
```

Comment:
I know this is a pretty dumb component but could the test be expanded a bit? Maybe one to verify userAvatar/defaultAvatar is correctl based off imageUrl in space?

Reply:
I have a test like this prepared, but I'm now stuck investigating why it causes PhantomJS to crash...